src/common/address-book/types.js

```diff
@@ -0,0 +1,17 @@
+const ADD_ADDRESS_BOOK_ENTRY = 'app/addressBook/ADD';
```

Comment:
type should be: `${SUBJECT}_${ACTION}`. In this case ADRESS_BOOK_ENTRY_ADD.
@rodrigopavezi  we discussed it, but we also need to document it somewhere